### DISCUSSION TITLE
letters in the Deutsche Bahn ticket (DB online ticket) are no longer displayed correctly

### DISCUSSION THREAD

**Discussion by seebronn** [sumatrapdfreader/sumatrapdf#2776] [Q&A]
for some time now, the letters in the Deutsche Bahn ticket (pdf DB online ticket) are no longer displayed correctly. Instead of umlauts, only placeholders are shown. How can I display German pdfs with correct umlauts?

**Comment by seebronn** [comment]
enclosed an example
![grafik](https://user-images.githubusercontent.com/50779703/176991355-64a8ce75-6da7-4fa2-867a-6e00abd0f33b.png)

**Reply by GitHubRulesOK** [reply]
It will be due to the Character Mapping and there can be generally two primary reasons
1) SumatraPDF in a few recent versions did not replace those characters efficiently (check your version number and try latest pre-release)
2) the tickets have the wrong mapping for widespread use (a sample would be needed to test and possibly allow for any fix)

**Comment by kjk** [comment]
This is fixed in latest 3.4.6 release
